docs/sources/troubleshooting/diagnostics.md

```diff
@@ -52,4 +53,4 @@ go tool trace <trace file>
 2019/11/24 22:20:42 Opening browser. Trace viewer is listening on http://127.0.0.1:39735
 ```
 
-See [Go command trace](https://golang.org/cmd/trace/) for more information about how to analyze trace files.
\ No newline at end of file
+See [Go command trace](https://golang.org/cmd/trace/) for more information about how to analyze trace files.
```

Comment:
```suggestion
For more information about how to analyze trace files, refer to [Go command trace](https://golang.org/cmd/trace/).
```

Reply:
I didn't really change this sentence but I like the suggestion 👍